packages/react/test/specs/lib/mergeThemes/mergeThemes-test.ts

```diff
@@ -387,4 +390,190 @@ describe('mergeThemes', () => {
+        },
+        'target',
+      )
+      const source = withDebugId(
+        {
+          siteVariables: { varA: 'sVarA' },
+          componentVariables: { Button: sv => ({ btnVar: sv.varA }) },
+          componentStyles: { Button: { root: ({ variables }) => ({ style: variables.btnVar }) } },
+        },
+        'source',
+      )
+
+      const merged = mergeThemes(target, source)
+
+      expect(merged.siteVariables).toMatchObject({
+        _debug: [
+          {
+            /* FIXME: unnecessary empty object */
+          },
+          { debugId: 'target' },
+          { debugId: 'source' },
+        ],
+      })
+
+      const buttonVariables = merged.componentVariables.Button(merged.siteVariables)
+      expect(buttonVariables).toMatchObject({
+        _debug: [
+          {
+            /* FIXME: unnecessary empty object */
+          },
+          { debugId: 'target' },
+          { debugId: 'source' },
+        ],
+      })
+
+      const buttonRootStyles = merged.componentStyles.Button.root({ variables: buttonVariables })
+      expect(buttonRootStyles).toMatchObject({
+        _debug: [{ debugId: 'target' }, { debugId: 'source' }],
+      })
+    })
+  })
+
+  // This test is disabled by default
+  // It's purpose is to be executed manually to measure performance of mergeThemes
+  xdescribe('performance', () => {
+    let originalDebugEnabled
+
+    beforeEach(() => {
+      originalDebugEnabled = debugEnabled.isEnabled
+    })
+
+    afterEach(() => {
+      Object.defineProperty(debugEnabled, 'isEnabled', {
+        get: () => originalDebugEnabled,
+      })
+    })
+
+    function mockIsDebugEnabled(enabled: boolean) {
+      Object.defineProperty(debugEnabled, 'isEnabled', {
+        get: jest.fn(() => enabled),
+      })
+    }
+
+    test('100 themes with debug disabled', () => {
+      mockIsDebugEnabled(false)
+
+      const merged = mergeThemes(..._.times(100, n => themes.teams))
+      const resolvedStyles = _.mapValues(
+        merged.componentStyles,
+        (componentStyle, componentName) => {
+          const compVariables = _.get(merged.componentVariables, componentName, callable({}))(
+            merged.siteVariables,
+          )
+          const styleParam: ComponentStyleFunctionParam = {
+            displayName: componentName,
+            props: {},
+            variables: compVariables,
+            theme: merged,
+            rtl: false,
+            disableAnimations: false,
+          }
+          return _.mapValues(componentStyle, (partStyle, partName) => {
+            if (partName === '_debug') {
+              // TODO: fix in code, happens only with mergeThemes(singleTheme)
+              return undefined
+            }
+            if (typeof partStyle !== 'function') {
+              console.log(componentName, partStyle, partName)
+            }
+            return partStyle(styleParam)
+          })
+        },
+      )
+      console.log(resolvedStyles.Button.root)
```

Comment:
Cruft UT?

Reply:
console.log removed, but I would like to have the test there (I run it manually to verify perf)